tests/test_image.py

```diff
@@ -0,0 +1,53 @@
+from morphocut import Pipeline
+from morphocut.image import ThresholdConst, Rescale, ImageWriter
+
+import numpy as np
+import skimage.io
+
+import pytest
+
+def test_ThresholdConst():
+    image = skimage.data.camera()
+    with Pipeline() as pipeline:
+        result = ThresholdConst(image, 256)
+
+    stream = pipeline.transform_stream()
+    pipeline.run()
+
+def test_Rescale():
+    image = skimage.data.camera()
+    with Pipeline() as pipeline:
+        result = Rescale(image, in_range=(0, 200), dtype=np.uint8)
+
+    stream = pipeline.transform_stream()
+    pipeline.run()
+
+#TODO: Confused in fmt and meta. 
+@pytest.mark.xfail(strict=True)
+def test_ImageWriter():
+    image_root = "/tests/images/"
+    fmt = "{ext}"
+    meta = ...
+    image = skimage.data.camera()
+    with Pipeline() as pipeline:
+        result = ImageWriter(image_root, fmt, image, meta)
```

Comment:
I replaced the outdated version of ImageWriter with the new one from io by moving io.ImageReader and io.ImageWriter to image. Please merge upstream/master.